internal/manifest/fixtures_info_test.go

```diff
@@ -0,0 +1,12 @@
+package manifest
```

Comment:
Non-blocking: Now that the files are in the Go-supported `testdata` subdirectory, this could probably be renamed to `testdata_info_test.go`.